types/vexflow/index.d.ts

```diff
@@ -65,6 +65,8 @@ declare namespace Vex {
         fillText(text : string, x : number, y : number) : IRenderContext;
         save() : IRenderContext;
         restore() : IRenderContext;
+        openGroup() : Node | void;
```

Comment:
`Node | void` doesn't make sense as a return type. Did you maybe mean `Node | undefined` ?

Reply:
I did this because of something I found weird in the library. For some implementations of the interface, this method is actually not supported but have an empty body thus returning `undefined` indeed. If this were the only implementation I think I would have put the return type `void` as it seemed more fitting so that's why I chose this return type. I'll do the change as requested but can you tell me if there is a reason other than semantic to use `Node | undefined` rather that `Node | void`. Maybe for avoiding to assign a variable to the void type ?

Reply:
Change pushed